awsutil/awsutil.go

```diff
@@ -56,6 +56,6 @@ func TimeDifference(server string) (time.Duration, error) {
 		return time.Duration(0), errors.New("invalid ntpdate output")
 	}
 
-	f, err := strconv.ParseFloat(string(submatched[1]), 64)
+	f, _ := strconv.ParseFloat(string(submatched[1]), 64)
```

Comment:
I understand, your change preserves the current behavior but makes it more explicit.

My question is why not simply alter the behavior to check the error instead of ignoring it? This is a question more to the maintainers of the library than to you, I'm not sure if there was a reason to not check the error when parsing float intentionally, or if it was an unintended omission.

If you look in the func code above, it checks errors when doing the other steps.